website/docs/sdks/index.md

```diff
@@ -76,6 +76,7 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
 | Basic support                                                                                     | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                          |
 | **Category: [Strategy constraints](../advanced/strategy_constraints)**                            |                        |                           |                    |                            |                        |                           |                      |                                                        |                                          |
 | Basic support (`IN`, `NOT_IN` operators)                                                          | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | ✅                                                     |                                          |
+| Advanced support (Semver, date, numeric and extended string operators)                            | ✅                     | ✅                        | ⭕                 | ⭕                         | ⭕                     | ⭕                        | ⭕                   | ⭕                                                     |                                          |
```

Comment:
There are a boat load of new operators so I've summarised them into things like Semver (even though there's a few semver operators). That means I haven't backticked them since they're not actual code artifacts, not sure how everyone feels about that

Reply:
Also, I would like to get PHP and Python released if possible (and this updated) before we approve this

Reply:
- PHP is released
- Node.js is released
- Proxy will be released tomorrow, just needs some validations.

Reply:
@sighphyre Love it! Sounds perfectly reasonable to not backtick them. If the proxy, PHP, and Node SDKs are out, can we update this PR and get them out? 🤩

Reply:
PHP and Node are in! I'll do another when the proxy happens (or Rust or Go for that matter)!

Reply:
@thomasheartman thanks! It's been done!